### DISCUSSION TITLE
Pro Account - Paypal

### DISCUSSION THREAD

**Discussion by pparvin** [Zefau/ioBroker.jarvis#1307] [General]
Hallo,

nachdem mein Pro-Account vom 09.12. nicht mehr funktioniert hat mußte ich erneut 12 Euro zahlen. Der Zeitraum hat sich dabei nicht verlängert sonder hat nun einen neuen End-Termin. Das finde ich nicht in Ordnung. Bitte beheben.
Wie kann ein bestehender Account wieder aktiviert werden?

Danke 
Patrik

<img width="276" alt="image" src="https://user-images.githubusercontent.com/8461116/146328023-f7a088b7-bc85-4089-b917-fb49e330fae5.png">

**Comment by Zefau** [comment]
Melde dich doch bei mir, bevor du erneut bezahlst. Der Zeitraum verlängert sich nicht, daher ist es ja ein Abo.

>  Wie kann ein bestehender Account wieder aktiviert werden?

Einfach die Lizenz aus deiner Rechnung in den Datenpunkt (nicht Objekt) `jarvis.0.info.pro` kopieren.

**Reply by Zefau** [reply]
Ich hab dir das Geld zurücküberwiesen. Bitte nutze die alte Lizenz. Wenn etwas nicht funktioniert, melde dich via ioBroker.jarvis@mailbox.org, ggf. mit der Fehlermeldung aus der Browser Konsole (F12).

**Reply by pparvin** [reply]
Ok, danke für die rasche Antwort.

**Comment by Zefau** [comment]
https://github.com/Zefau/ioBroker.jarvis/discussions/891

**Comment by CLAM01** [comment]
Hallo zusammen, ich habe mir das Jarvis 3.0.13 auch mal angeschaut. Sehr schön.
Nun wollte ich mich mal anmelden über den PayPal Knopf.
Wenn ich mich angemeldet habe, dann geht das PayPal Fenster weg (wohin auch immer und was da auch immer nun im Hintergrund passiert).

Ein Login hat trotzdem nicht stattgefunden, denn ich sehe dann weiterhin nur das ich nicht eingeloggt bin.
So ein Verhalten der Software stimmt mich dann etwas nachdenklich, gerade weil ich aus einem hochsensiblen Bereich komme, wo Daten sehr sensibel behandelt werden müssen und die maximale transparenz da sein muss.

Aber ich denke, dass ich erst mal warte, bis es eine ausgereifte Version gibt.
Ich lese hier viel, das dies und das nicht geht, Lizenzen nicht übertragen werden können etc.
In Anbetracht der derzeitigen Inflation sind mir da gerade auch 12 Euro zu viel. :-)

Zerfau, nicht böse gemeint, die Software ist cool und ich nutze bereits die Version 2.x

**Comment by Zefau** [comment]
PayPal hat die API geändert, daher funktioniert es gerade nicht. Der Fix kommt zeitnah.

>  SO ein Verhalten der Software stimmt mich dann etwas nachdenklich, gerade weil ich aus einem hochsensiblen BEreich komme, wo Daten sehr sensibel behandelt werden müssen und die maximale transparenz da sein muss.

SO ein Verhalten kann immer SO viele Gründe haben. Am Ende ist es ein User-Projekt von dem du nicht die Erwartungshaltung einer professionellen Software haben kannst.

>  Ich lese hier viel, das dies und das nicht geht, Lizenzen nicht übertragen werden können etc.

Wo liest du was? Da musst du aufpassen, ob dies die v3.0 betrifft oder eher die v3.1-alpha.

>  In Anbetracht der derzeitigen Inflation sind mir da gerade auch 12 Euro zu viel. :-)

Umso höher die Inflation umso weniger sind die 12 EUR wert. Insofern sind sie in jarvis gut investiert.

**Reply by CLAM01** [reply]
Sorry, SO sollte So heißen. Das ist eine Schwäche von mir, dass ich immer zwei Großbuchstaben schreibe. Korrigiere ich gerade mal, denn das war nicht schroff gemeint. :-)

**Reply by CLAM01** [reply]
_PayPal hat die API geändert, daher funktioniert es gerade nicht. Der Fix kommt zeitnah._

Wird die Version dann auch in IOBroker angeboten? Da ist ja bisher nur die 3.0.13 oder muss ich dann über einen Github Link installieren?

**Reply by Zefau** [reply]
Der Fix ist in v3.1-alpha bereits verfügbar und kommt auch zeitnah in die v3.0.x

**Reply by CLAM01** [reply]
Sorry, ich bin heute zu blöd die 3.1. alpha 5x zu instalieren.
IO Broker rödelt immer nur rum wenn ich es von Github installieren möchte und jetzt mag es gar nicht mehr auf meinem Testsystem :-)

**Reply by mcuiobroker** [reply]
npm-Version?
nodejs-Version?

**Reply by CLAM01** [reply]
jo, ich depp... :-)
Ne, auf NPM sind ja immer nur die freigegebenen Versionen. Ich will aber die noch nicht offical Version 3.1 alpha haben.
Vom NPM hat er mir jetzt wieder ne 3.0.xx gezogen.
Ich gehe nur auf die Octocat und sage dann Jarvis von GIT holen. Aber da kommt nix...

Außer:
$ iobroker url Zefau/ioBroker.jarvis --host iobroker
install Zefau/ioBroker.jarvis
NPM version: 6.14.17Installing Zefau/ioBroker.jarvis... (System call)

**Reply by mcuiobroker** [reply]
Welche npm -Version ist auf dem System -> console -> `npm -v`
Weil es bei npm -v -> v6.x.x manchmal ca 15 minuten dauert bis das Upgrade gezogen wird.

**Reply by mcuiobroker** [reply]
D.h. du solltest mal Dein Testsystem updaten, siehe Beschreibung in der Doku oder von Thomas Braun.

**Reply by CLAM01** [reply]
Hi, das schaue ich mir mal an. Habe das eigendlich erst vor kurzem installiert (IOBroker).
Na dann schaue ich mal nach der NPM Version.
Danke dir schon mal.

BTW: Ich habe mir mal die Roadmap angeschaut (Als PO/PM ist man ja immer an sowas ineteressiert). Da sind mir direkt die Gauges und Balken ins Auge geschossen. Das finde ich klasse, denn das vermisse ich total, das man sich Gauges anzeigen lassen kann etc..

**Reply by mcuiobroker** [reply]
https://forum.iobroker.net/topic/35090/howto-nodejs-installation-und-upgrades-unter-debian
Version sieht man bei der Installation:
![image](https://user-images.githubusercontent.com/84818934/194950878-1e80f5d7-cd6c-4221-b5bd-97eebe118044.png)

**Reply by mcuiobroker** [reply]
https://forum.iobroker.net/topic/44566/how-to-node-js-iobroker-richtig-updaten-2021-22-edition

**Reply by CLAM01** [reply]
Danke dir, node.js ist bei mir schon auf 18, da ich sonst Jarvis 3 nicht hätte drauf machen können weil ich da noch auf 12 war.

Nun bekomme ich nur das hier angezeigt im ProAccount Tab.
Console spuckt nichts auffälliges aus im Browser.

![image](https://user-images.githubusercontent.com/115494634/195010139-9fab7f38-4bb0-4aac-aa6e-036fafca970e.png)

**Reply by mcuiobroker** [reply]
V18 ist nicht freigegeben und wird nicht VPN den Adaptern unterstützt. V16 nutzen.

**Reply by CLAM01** [reply]
VPN nutze ich ja nicht. Aber ich kann gerne mal downgraden auf 16 und schauen ob dann die PRO Seite geht.

**Reply by mcuiobroker** [reply]
VPN sollte von heißen,  Autokorrektur vom Handy?

**Reply by CLAM01** [reply]
So, jetzt V16.17.1 drauf, aber in der PRo seite kann ich weiterhin nichts mit PayPal machen.

**Reply by mcuiobroker** [reply]
Web-Einstellungen? Cache aktiviert? System nach dem Update komplett neu gestartet?
Cache vom Browser löschen.

**Reply by CLAM01** [reply]
> Web-Einstellungen? Cache aktiviert? System nach dem Update komplett neu gestartet? Cache vom Browser löschen.

Web-Einstellung nicht anders als bei 3.0.x
Cache aktiviert - KA
System reboot (Host) - ja
Browsercache gelöscht - ja

**Reply by mcuiobroker** [reply]
![image](https://user-images.githubusercontent.com/84818934/195087739-e5ac688c-8433-4214-abb4-21a54de3bc1a.png)

Dann nochmal auf der console: `iob upload jarvis`

Dreht sich beim Lizenz abrufen der Kreis?
![image](https://user-images.githubusercontent.com/84818934/195091820-5ad47acc-9179-4e4e-82bd-487123f6b509.png)
